graphql/project_resolver.go

```diff
@@ -52,6 +52,11 @@ func (r *projectResolver) Patches(ctx context.Context, obj *restModel.APIProject
 	return &Patches{Patches: apiPatches, FilteredPatchCount: count}, nil
 }
 
+// Private is the resolver for the private field.
+func (r *projectResolver) Private(ctx context.Context, obj *restModel.APIProjectRef) (*bool, error) {
+	panic(fmt.Errorf("not implemented: Private - private"))
```

Comment:
This will panic every time a user queries this. So you would need to remove this from `project.graphql` first. (But also check to make sure its not being used on the UI)